Document changes to participant ID requirements (#108)
AlexsLemonade/scpca-docs: docs/download_files.md

| `technology`      | 10X kit used to process library                                |
| `filtered_cell_count` | Number of cells after filtering with `emptyDrops`          |
| `submitter_id`    | Original sample identifier from submitter                      |

> | `participant_id`  | Original participant id, required to be present if there are multiple samples from the same participant, optional for all other samples                                                                        |

Reviewer: ```suggestion
| `participant_id`  | Original participant id, required when there are multiple samples from the same participant, optional for all other samples                                                                        |
```
